Packs/FortiSIEM/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,33 @@
+#### Incident Fields
+- **FortiSIEM Incident Last Seen**
+- **FortiSIEM Incident First Seen**
```

Comment:
@yuvalbenshalom I went through all of them, some were changed and some weren't since they were either unique or their type isn't the same as the current fields, like `firstseen` which is `shortText` and here it's `timestamp`.